### PR TITLE
RELEASE-PROCESS.MD: make warning about required template changes more prominent

### PR DESCRIPTION
**Pull request by chalin**
In followup to these two issues:

- #2809
- #2810 

this PR updates the release-process notes to:

- Makes more prominent the side-note that reminds the release developer to update the VERSION in the template
- Removes the angle brackets around `<INSERT-CORRECT-VERSION>` because that makes the text disappear when the markdown is rendered/previewed -- since the renderer considers it to be a HTML tag. (It might be a good idea to remove all angle brackets from that template)

/cc @tomkerkhove

### CODE REVIEW

File: RELEASE-PROCESS.MD [hunk 1]
```diff
@@ -33,14 +36,19 @@ KEDA Deployment YAML file (eg. keda-2.4.0.yaml) is also automatically created an
...
-Every release should use the provided template below to create the GitHub release and ensure that a new GitHub Discussion is created.
+Every release should use the template provided below to create the GitHub release and ensure that a new GitHub Discussion is created.
 
-> 💡 Don't forget to update the version & new contributors in the template
+> ### 💡 IMPORTANT
+> 
+> Remember to make the following changes to the template:
+> 
+> - Replace `INSERT-CORRECT-VERSION` (there are **two** occurrences in the template) with the new-release ID
+> - Update the list of new contributors
```
Comment (tomkerkhove) [inline RELEASE-PROCESS.MD:46]: Can you give some info here or is it in the README for docs?

**Review by tomkerkhove — APPROVED** [review]
LGTM

**Review by JorTurFer — APPROVED** [review]

File: RELEASE-PROCESS.MD (hunk 1 above)
Comment (chalin) [reply RELEASE-PROCESS.MD:46]: I don't know. This was part of your original release-process text, I just reformatted it. Can I leave this up to y'all to update in this or a followup PR?

Comment (zroubalik) [reply RELEASE-PROCESS.MD:46]: I think it is fine

**Review by zroubalik — APPROVED** [review]

**Comment by zroubalik** [comment]
@chalin seems like static checks are failing on some formatting issue, could you please look at it?

**Comment by chalin** [comment]
All checks are passing now ✅

**Pull request merged**